### PR TITLE
add Atom package to Editors Readme

### PR DESCRIPTION
**Pull request by aminya**

### CODE REVIEW

File: Editors/README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ After installing the extension, settings for SourceKit-LSP can be found in `Pref
 * `sourcekit-lsp.toolchainPath`: (optional) The path of the swift toolchain (sets `SOURCEKIT_TOOLCHAIN_PATH`). By default, sourcekit-lsp uses the toolchain it is installed in.
 * `sourcekit-lsp.tracing.server`: Traces the communication between VS Code and the SourceKit-LSP language server
 
+## Atom
+
+Download the ide-sourcekit package for Atom from [this download link](https://atom.io/packages/ide-sourcekit). The installation instructions are given the same link.
```
Comment (MaxDesiatov) [inline Editors/README.md:34]: Just a slight wording suggestion.

```suggestion
Download the `ide-sourcekit` package for Atom from [a corresponding package page](https://atom.io/packages/ide-sourcekit). It also contains installation instructions to get you started.
```

**Comment by MaxDesiatov** [comment]
@swift-ci please test

File: Editors/README.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ After installing the extension, settings for SourceKit-LSP can be found in `Pref
 * `sourcekit-lsp.toolchainPath`: (optional) The path of the swift toolchain (sets `SOURCEKIT_TOOLCHAIN_PATH`). By default, sourcekit-lsp uses the toolchain it is installed in.
 * `sourcekit-lsp.tracing.server`: Traces the communication between VS Code and the SourceKit-LSP language server
 
+## Atom
+
+Download the `ide-sourcekit` package for Atom from [a corresponding package page](https://atom.io/packages/ide-sourcekit). It also contains installation instructions to get you started.
```
Comment (benlangmuir) [inline Editors/README.md:34]: Gramatically it should be "**the** corresponding package page". Otherwise LGTM

Comment (aminya) [reply Editors/README.md:34]: I wanted to change it. But Github's suggestion mechanism did not allow me.

Comment (MaxDesiatov) [reply Editors/README.md]: My bad, thanks for the correction 🙏

**Comment by MaxDesiatov** [comment]
@swift-ci please test

**Review by MaxDesiatov — APPROVED** [review]

**Comment by MaxDesiatov** [comment]
@swift-ci please test

**Comment by benlangmuir** [comment]
@swift-ci please test

**Comment by benlangmuir** [comment]
@swift-ci please test Linux

**Pull request merged**